curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c151b23bf21bc7c4fcba.md

```diff
@@ -7,7 +7,13 @@ dashedName: step-67
 
 # --description--
 
-Finally, update the `text.innerText` assignment to equal the `text` from the `location` object. However, instead of using bracket notation, use <dfn>dot notation</dfn>. Here is an example of accessing the `name` property of an object called `obj`: `obj.name`.
+Finally, update the `text.innerText` assignment to equal the `text` from the `location` object.However, instead of using bracket notation, use <dfn>dot notation</dfn>.
```

Comment:
```suggestion
Finally, update the `text.innerText` assignment to equal the `text` from the `location` object. However, instead of using bracket notation, use <dfn>dot notation</dfn>.
```